### ISSUE TITLE
npm "Could not resolve dependency" with Angular 17

### ISSUE THREAD

**Issue by Mr-Vinti** [jtc10005/ngx-flex-layout#5]
# Bug Report

#### What is the expected behavior?
To install ngx-flex-layout via npm without issues
This is a copy of #4 , as Angular just launched v17

#### What is the current behavior?
```
ng update @angular/material @angular/cdk
Using package manager: npm
Collecting installed dependencies...
Found 36 dependencies.
Fetching dependency metadata from registry...
                  Package "ngx-flexible-layout" has an incompatible peer dependency to "@angular/cdk" (requires "^16.0.0", would install "17.0.0").
✖ Migration failed: Incompatible peer dependencies found.
Peer dependency warnings when installing dependencies means that those dependencies might not work correctly together.
You can use the '--force' option to ignore incompatible peer dependencies and instead address these warnings later.
  See "/private/var/folders/2w/_w5l0jgx4_q424nflbp9m458bjbhpc/T/ng-FdI4m4/angular-errors.log" for further details.
```

#### What are the steps to reproduce?
Try to install the module on a project with v17 or try to upgrade a v16 project to v17

#### What is the use-case or motivation for changing an existing behavior?
N/A

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 17

#### Is there anything else we should know?
N/A

**Comment by jtc10005** [comment]
starting this now

**Comment by jtc10005** [comment]
should be resolved now in #6.  @Mr-Vinti let me know if you see any issues.
